pkgs/tools/security/ossec/no-root.patch

```diff
@@ -139,11 +139,11 @@ diff -Naur ossec-hids-2.6-orig/src/InstallAgent.sh ossec-hids-2.6/src/InstallAge
 +echo chmod 700 ${DIR}/.ssh
  
  
- # For the /var/run
--chmod 770 ${DIR}/var/run
--chown root:${GROUP} ${DIR}/var/run
-+echo chmod 770 ${DIR}/var/run
-+echo chown root:${GROUP} ${DIR}/var/run
+ # For the /run
+-chmod 770 ${DIR}/run
```

Comment:
it is a ```.patch``` file and the lines to be removed on the patch apply.
changing them means the patch apply will fail